utils/cpupower/Makefile

```diff
@@ -0,0 +1,48 @@
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/kernel.mk
+
+PKG_NAME:=cpupower
+PKG_VERSION:=$(LINUX_VERSION)
+PKG_RELEASE:=1
+
+PKG_MAINTAINER:=John Audia <therealgraysky@proton.me>
+PKG_LICENSE:=GPL-2.0-only
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/cpupower
+  SECTION:=utils
+  CATEGORY:=Utils
+  TITLE:=Shows and sets processor power related values
+  URL:=https://www.kernel.org
+  VERSION:=$(LINUX_VERSION).$(PKG_RELEASE)
```

Comment:
I'm so sorry, I overlooked this line. It should be dropped since it is generated automatically from PKG_VERSION and PKG_RELEASE.
Thanks for your contribution and patience.

Reply:
Done!